packages/components/src/components/select/select.stories.ts

```diff
@@ -71,600 +61,314 @@ export default {
+        <sd-option value="option-4">Option 4</sd-option>
+        <sd-option value="option-5">Option 5</sd-option>
+      </sd-select>
+    </div>
+  `
 };
 
 /**
- * `sd-select` is fully accessibile via keyboard.
+ * Use the `multiple` attribute to allow multiple options to be selected.
  */
 
-export const Mouseless = {
-  parameters: {
-    controls: {
-      exclude: ['open-attr', 'value', 'default']
-    }
-  },
-  render: (args: { 'open-attr'?: string }) => {
-    delete args['open-attr'];
-
-    return html`<div class="mouseless h-[260px] w-full flex gap-4">
-      ${generateTemplate({
-        constants: [twoOptionsConstant, { type: 'attribute', name: 'label', value: 'Default' }],
-        args
-      })}
-      ${generateTemplate({
-        constants: [twoOptionsConstant, multipleConstant, { type: 'attribute', name: 'label', value: 'Multiple' }],
-        args
-      })}
-      ${generateTemplate({
-        constants: [
-          twoOptionsConstant,
-          multipleConstant,
-          { type: 'attribute', name: 'useTags', value: true },
-          { type: 'attribute', name: 'label', value: 'Multiple w/ Tags' }
-        ],
-        args
-      })}
-    </div>`;
-  },
+export const Multiple = {
+  render: () => html`
+    <div class="w-[400px] h-[500px]">
+      <sd-select size="lg" label="Label" placement="bottom" multiple="" value="option-1 option-2">
+        <sd-option value="option-1">Option 1</sd-option>
+        <sd-option value="option-2">Option 2</sd-option>
+        <sd-option value="option-3">Option 3</sd-option>
+        <sd-option value="option-4">Option 4</sd-option>
+        <sd-option value="option-5">Option 5</sd-option>
+      </sd-select>
+    </div>
+  `
+};
 
-  play: async ({ canvasElement }: { canvasElement: HTMLUnknownElement }) => {
-    const el = canvasElement.querySelector('.mouseless sd-select');
-    await waitUntil(() => el?.shadowRoot?.querySelector('input'));
-    el?.shadowRoot?.querySelector('input')!.focus();
-  }
+/**
+ * Use the `useTags` attribute to display selected options as tags using the `sd-tag` component.
+ *
+ * **Hint:** it requires the `multiple` attribute to be set.
+ */
+
+export const useTags = {
+  render: () => html`
+    <div class="w-[400px] h-[500px]">
+      <sd-select size="lg" label="Label" placement="bottom" multiple="" value="option-1 option-2" useTags>
+        <sd-option value="option-1">Option 1</sd-option>
+        <sd-option value="option-2">Option 2</sd-option>
+        <sd-option value="option-3">Option 3</sd-option>
+        <sd-option value="option-4">Option 4</sd-option>
+        <sd-option value="option-5">Option 5</sd-option>
+      </sd-select>
+    </div>
+  `
 };
 
 /**
- * Use `<sl-divider>` to group listbox items visually. You can also use `<small>` to provide labels, but they won’t be announced by most assistive devices.
+ * Use the `max-options-visible` attribute to define the maximum number of selected options that will be visible.
```

Comment:
On the figma sample the "+1" also has a closing option. Can we have it? Or do we need a ticket to fix this?

Reply:
Let's discuss it tomorrow in the daily, I'm not sure about this one.